MODULE.bazel

```diff
@@ -30,6 +30,16 @@ register_toolchains(
     "@rules_perl//perl:perl_windows_x86_64_toolchain",
 )
 
+cpan = use_extension("@rules_perl//perl/cpan:extensions.bzl", "perl_cpan")
+cpan.install(
+    name = "rules_perl_deps",
```

Comment:
This kinda may make it seem like these are for everyone to use. Maybe call it `cpan_compiler_deps`?

Reply:
Done!